### PR TITLE
Gui/settings: fix initialize value for news config.

### PR DESCRIPTION
**Pull request by Zangetsu38**
fix issue with get wrong default value on news config file created.

### CODE REVIEW

File: src/emulator/host/include/host/config.h [hunk 1]
```diff
@@ -40,11 +40,11 @@ struct Config {
...
-    std::string pref_path;
+    std::string pref_path = {};
     bool archive_log = false;
     bool texture_cache = true;
     optional<bool> wait_for_debugger;
-    std::string background_image;
-    float background_alpha;
-    int icon_size;
+    std::string background_image = {};
+    float background_alpha = 0.000f;
```
Comment (pent0) [inline src/emulator/host/include/host/config.h]: Set default alpha to something that people can see (in mid of not visible and totally visible, 0.5f or 129.0f? I'm not sure)

File: src/emulator/config.cpp [hunk 2]
```diff
@@ -182,12 +182,16 @@ InitResult init(Config &cfg, int argc, char **argv) {
         po::options_description config_desc("Configuration");
         config_desc.add_options()
             ("archive-log,A", po::bool_switch(&cfg.archive_log), "Makes a duplicate of the log file with TITLE_ID and Game ID as title")
+            ("background-alpha,a", po::value(&cfg.background_alpha), "Background Alpha")
```
Comment (pent0) [inline src/emulator/config.cpp]: I don't think people will specifiy these stuffs in command line, also they doesn't have debug value. Delete those changes (except log_uniforms).

File: src/emulator/host/include/host/config.h (hunk 1 above)
Comment (Zangetsu38) [reply src/emulator/host/include/host/config.h]: value is already set by default on 0.000f, and this value is for full transparency, but yes, you have right, for game i recommande 0.300 for see game title correctly

Comment (pent0) [reply src/emulator/host/include/host/config.h]: Yeah

File: src/emulator/config.cpp (hunk 2 above)
Comment (Zangetsu38) [reply src/emulator/config.cpp]: oki, i have delete

File: src/emulator/host/include/host/config.h (hunk 1 above)
Comment (Zangetsu38) [reply src/emulator/host/include/host/config.h]: fixed so

**Pull request merged**